src/app/components/elements/list-groups/ListGroupFooterBottom.tsx

```diff
@@ -20,18 +57,10 @@ export const ListGroupFooterBottom = () => (
 
       <ListGroupItem className="footer-bottom-logos border-0 px-0 py-0 pb-4 pb-md-1 bg-transparent d-flex justify-content-between d-print-none">
         <ExternalLink href="https://computingeducation.org.uk/">
-          <img
-            src="/assets/logos/ncce.svg"
-            alt="National Centre for Computing Education website"
-            className="logo-mr"
-            height="57px"
-          />
-        </ExternalLink>
-        <ExternalLink href="https://www.stem.org.uk/">
-          <img src="/assets/logos/stem.svg" alt="STEM Learning" className="logo-mr" height="57px" />
+          <img src="/assets/logos/ncce.svg" alt="National Centre for Computing Education website" className="logo-mr" />
         </ExternalLink>
         <ExternalLink href="https://www.gov.uk/government/organisations/department-for-education">
-          <img src="/assets/logos/dfe.svg" alt="UK Department for Education" className="logo-mr" height="57px" />
+          <img src="/assets/logos/dfe.svg" alt="UK Department for Education" className="logo-mr" />
```

Comment:
I'd double check the styling of these images two here, they should be aligned left exactly next to each other, but they seem to be centered. It's most obvious on mobile